src/app/clusters/ota-requestor/DefaultOTARequestorDriver.cpp

```diff
@@ -470,5 +509,45 @@ bool DefaultOTARequestorDriver::GetNextProviderLocation(ProviderLocationType & p
     return false;
 }
 
+CHIP_ERROR DefaultOTARequestorDriver::ScheduleRetry(bool trySameProvider)
```

Comment:
nit: Might be helpful to call this `ScheduleQueryRetry` to explicitly say what action we are retrying

Reply:
Renamed to ScheduleQueryRetry().